cmd/ci-secret-mirroring/README.md

```diff
@@ -0,0 +1,50 @@
+# CI-Secret-Mirroring
+
+This tool extends the [populate-secrets-from-bitwarden.sh](https://github.com/openshift/release/blob/c8c89d08c56c653b91eb8c7580657f7ce522253f/ci-operator/populate-secrets-from-bitwarden.sh)
+to support mirroring secrets cross Kubernetes/OpenShift-clusters.
+
+We use `--kubeconfig` to specify the path to a [kube config](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
+that the tool will load and use it to access clusters for writing secrets.
+
+It expects a configuration like the one below which specifies the mapping from the items
+in BitWarden and the targeting secret.
+
+```yaml
+secrets:
+- from:
+    - bw-item: item-name-1
+      - field: field-name-1
+      - field: field-name-2
+      - attachment: attachment-name-1
+    - bw-item: item-name-2
+      - field: field-name-3
+      - attachment: attachment-name-2
+      - attachment: attachment-name-3
+  to:
```

Comment:
Why don't we re-use the Prow `context` names here? Expect the `--kubeconfig` to have namespace-specific credentials by context

Reply:
Thought about this too.

Do you mean for each `namespace` we have a `context` in `kubeconfig`?

Current context is cluster level, it can write secrets to all related namespaces.

If the context is ns level, we would need more and more contexts in the kubeconfig. Do we want to do that?

Reply:
A context in the kubeconfig _is_ namespaced.